package.json

```diff
@@ -167,6 +168,9 @@
     "roots": [
       "<rootDir>/packages",
       "<rootDir>/scripts"
+    ],
+    "transformIgnorePatterns": [
+      "node_modules/(?!@novnc)"
```

Comment:
this needs to be added to the one on line 165.  Looks like a rebase issue.
```
 "transformIgnorePatterns": [
       "node_modules/(?!@patternfly)",
       "node_modules/(?!@novnc)"
  ],
```

Reply:
That should fix the build

Reply:
right, a rebase issue.
Thanks for spotting this.